task/host-sp-comms/src/bsp/gimlet_bcde.rs

```diff
@@ -593,7 +593,7 @@ impl ServerImpl {
         let packrat = &self.packrat; // partial borrow
         let mut data = InventoryData::DimmSpd {
             id: [0u8; 512],
-            temp_sensor: SensorId(u32::MAX),
+            temp_sensor: SensorId::INVALID,
```

Comment:
Per my concerns about `SensorId::INVALID` below, I think we could populate this directly with `i2c_config::sensors::TSE2004AV_TEMPERATURE_SENSORS[index as usize]` here.